src/css/_components.scss

```diff
@@ -1,16 +1,16 @@
-@import 'components/settings';
-@import 'components/material-switch';
-@import 'components/toolbar';
-@import 'components/minor-floating-action-button';
-@import 'components/floating-action-button';
-@import 'components/spinner';
-@import 'components/toasts';
-@import 'components/material-slider';
-@import 'components/drop-overlay';
-@import 'components/changelog';
-@import 'components/svg-output';
-@import 'components/material-tabs';
-@import 'components/results';
-@import 'components/output';
-@import 'components/code-output';
-@import 'components/ripple';
+@import "components/settings";
```

Comment:
How do you feel about sticking with single quotes like the JS? One key seems better than two 😄

Reply:
I personally prefer double quotes in CSS/Sass, but it's your call, I can switch them.

BTW in the past I've hit a pretty nasty IE bug, where using single quotes in a data URI broke IE from rendering.

Reply:
https://github.com/mdo/github-buttons/commit/db9e024f389df3e92b6c067becfec32953495048